src/js/panels.js

```diff
@@ -1090,8 +1104,28 @@ https://github.com/fluid-project/first-discovery/raw/master/LICENSE.txt
                 "this": "{that}.dom.token",
                 method: "html",
                 args: ["{arguments}.0"]
+            },
+            savePrefsToServer: {
+                funcName: "gpii.firstDiscovery.panel.token.savePrefsToServer",
+                args: ["{that}", "{that}.options.saveRequestConfig", "{that}.data"]
             }
         }
     });
 
+    gpii.firstDiscovery.panel.token.savePrefsToServer = function (that, saveRequestConfig, data) {
+        $.ajax({
+            url: saveRequestConfig.url,
+            method: saveRequestConfig.method,
+            contentType: "application/json",
+            dataType: "json",
+            data: data,
+            success: function (data) {
+                that.events.onSuccess.fire(data.token);
+            },
+            error: function () {
+                that.events.onError.fire();
+            }
```

Comment:
probably better to return all the data from the success and error and handle it as needed in the listener.